src/feature/user-profile/card-password.tsx

```diff
@@ -77,22 +93,39 @@ export default function PasswordCard() {
         <CardContent className="space-y-2">
           <div className="space-y-1">
             <Label htmlFor="current">Current password</Label>
-            <Input
-              id="current"
-              type="password"
-              disabled={!isEditing}
-              {...register('currentPassword', { required: 'Current password is required' })}
-            />
-            {errors.currentPassword && (
-              <p className="text-sm text-red-500">{errors.currentPassword.message}</p>
-            )}
+            <div className="relative flex items-center">
+              <Input
+                id="current"
+                placeholder="••••••••"
+                type={isPasswordVisible ? 'text' : 'password'}
+                disabled={!isEditing}
```

Comment:
Предлагаю убрать плейсхолдер, он сбивает с толку
```suggestion
                id="current"
                type={isPasswordVisible ? 'text' : 'password'}
                disabled={!isEditing}
```

Reply:
согласна, убрала